tests/testthat/test-make_mock_data.R

```diff
@@ -199,3 +199,36 @@ test_that("Check mock when value it missing", {
 })
 
 
+test_that("Test when no body_style is present", {
+  tfrmt_obj_one_span <- tfrmt(
+    label = label,
+    group = group,
+    param = param,
+    column = vars(columns),
+    col_plan = col_plan(
+      group,label,
```

Comment:
if these don't get specified here, do they end up last?

Reply:
well technically last in the dataset. But because of how gt works it will automatically pull the labels to the left. I don't think you could have grouping and labels on the right hand side